lib/utilities/monitoringHandler.js

```diff
@@ -169,21 +179,21 @@ function routeHandler(req, res, log, cb) {
  */
 function monitoringHandler(clientIP, req, res, log) {
     function monitoringEndHandler(err, results) {
-        writeResponse(res, err, log, results, error => {
+        writeResponse(res, err, results, error => {
```

Comment:
Is this function called with a `results` parameter that is not equal to `[]`? The `routeHandler` function never calls this function if there is no error.

Reply:
correct, I don't think it is really used at the moment.
I don't know why it's there, but there may be a reason why it was added, i'd prefer not rushing to clean this thing..